### ISSUE TITLE
Page titles

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/meet#1047]
## Bug Report

**Problematic behavior**
Page titles are not relevant, users cannot distinguish between the two "Visio" tabs, the first page being the Visio presentation site page, the second being the page for the meeting we are connected to.

**Expected behavior/code**
`<title>Visio</title>`

**Possible Solution**
The Visio site homepage should display "Visio - Home"

The meeting connection page must reflect the content of this page: "Visio - Join Meeting"

**Additional context/Screenshots**
<img width="489" height="58" alt="Image" src="https://github.com/user-attachments/assets/6bb3b1d6-49c5-4d81-b315-c58982132b5f" />
